src/node_contextify.cc

```diff
@@ -261,7 +264,7 @@ class ContextifyContext {
     Environment* env = Environment::GetCurrent(args);
     if (debug_context.IsEmpty()) {
       // Force-load the debug context.
-      Debug::GetMirror(args.GetIsolate()->GetCurrentContext(), args[0]);
+      Debug::SetDebugEventListener(args.GetIsolate(), DummyDebugEventListener);
```

Comment:
You might as well use a lambda to put things closer together:
```c++
auto dummy_event_listener = [] (const Debug::EventDetails&) {};
Debug::SetDebugEventListener(args.GetIsolate(), dummy_event_listener);
```
Not a deal-breaker, though.